### PR TITLE
Update introductory copy

### PR DESCRIPTION
**Pull request by kytrinyx**
This adds a 'blurb' to the config file, and tweaks the `about` doc.

I really like the sentence in the about document that talks about how gophers like to describe the language. It feels very friendly and colloquial, and makes a great hook to pull people in to read more.

So I put that first, and then tweaked the second sentence simply for flow, so that it fit after the opening paragraph.

See https://github.com/exercism/discussions/issues/173

/cc @iHiD @ErikSchierboom

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "language": "Go",
   "active": true,
+  "blurb": "Go is a compiled, open source programming language with a small, consistent syntax, a powerful standard library and fantastic tooling. It's a great fit for web backends and command-line tools.",
```
Comment (robphoenix) [inline config.json]: At risk of getting too long and list-y, maybe mention systems & concurrency?

*"Go is a compiled, open source programming language with a small, consistent syntax, a powerful standard library, built-in concurrency primitives and fantastic tooling. It's a great fit for web backends, command-line tools and large-scale systems."*

**Comment by robphoenix** [comment]
Do it @ErikSchierboom! I can recommend some GREAT exercises to help you learn. :wink:

**Comment by ErikSchierboom** [comment]
@robphoenix I just might :)

File: config.json (hunk 1 above)
Comment (kytrinyx) [inline config.json]: The current blurb is already at the word limit. The 'about' is going to show farther down the page, so as long as we're happy with how the about doc expands on the blurb, I think we should be good with the shorter one.

**Comment by kytrinyx** [comment]
> I would suggest adding that bit about tooling from the blurb to the ABOUT right there at the end.

That's a great point @tleen. I think we should actually expand on it a little bit in the about, rather than keep it the same. I'll take a stab at that.

**Comment by kytrinyx** [comment]
Alright, I made a tweak to the ABOUT doc to mention the tooling. Not quite sure about the wording there.

File: docs/ABOUT.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 
-Go is noted for the [concurrent programming features](https://golang.org/doc/effective_go.html#concurrency) built into the language core, the networking packages in the standard library ([such as a web server](https://golang.org/pkg/net/http/)), fast compilation and execution speed, and its simple, minimalistic language design.
+The language is open source, and was started by engineers at Google. It's written using a C-style syntax, has statically typed variables, manages memory using garbage collection, and is compiled into stand-alone executables.
 
-Fans of Go (called *gophers*) describe it as having the expressiveness of dynamic languages like Python or Ruby, with the performance of compiled languages like C or C++.
+Go is noted for the [concurrent programming features](https://golang.org/doc/effective_go.html#concurrency) built into the language core, the networking packages in the standard library ([such as a web server](https://golang.org/pkg/net/http/)), fast compilation and execution speed, and its simple, minimalistic language design. The consistency of the language makes its delightful and abundant tooling possible.
```
Comment (tleen) [inline docs/ABOUT.md]: Perhaps it needs to be broken up differently. Something like:

> execution speed. Its simple, minimalistic and consistent language design make for a delightful experience. While the abundant and thoughtful provided tooling addresses traditional problems such as [consistent formatting](https://godoc.org/github.com/golang/go/src/cmd/gofmt) and [documentation](http://godoc.org/golang.org/x/tools/cmd/godoc).

Comment (kytrinyx) [inline docs/ABOUT.md]: Yes! That's so much better. I'm taking it.

**Comment by kytrinyx** [comment]
I'm going to go ahead and merge this as it seems like we have several approvals. If there are further tweaks let's do it in a new PR.

Thanks for your help, everyone! 🌷

**Pull request merged**